.theia/launch.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
you will need to remove this folder using `git rm -r -f . theia` so git can forget them